tests/probes/rpm/rpm_common.sh

```diff
@@ -11,10 +11,7 @@ RPMBUILD="${RPMBASE}/build"
 
 # Since Fedora 36 RPM database location changed, see
 # https://fedoraproject.org/wiki/Changes/RelocateRPMToUsr
-# However, /var/lib/rpm/ still works as it is a symlink to
-# the new path, /usr/lib/sysimage/rpm/, in Fedora >= 36
-# Therefore, always use /var/lib/rpm/ as it always works.
-RPMDB_PATH="/var/lib/rpm/"
+RPMDB_PATH="/usr/lib/sysimage/rpm/"
```

Comment:
We still need to use `/var/lib/rpm/` here because, for example, on CentOS Stream 9:
```
# ls /usr/lib/sysimage/rpm/
ls: cannot access '/usr/lib/sysimage/rpm/': No such file or directory
```

Reply:
Or maybe we can do similar as in openscap source, check if `/usr/lib/sysimage/rpm/` exists, if not use `/var/lib/rpm/`.